src/applications/personalization/profile/tests/e2e/direct-deposit/display-consistently/direct.deposit.connecttion.error.cypress.spec.js

```diff
@@ -0,0 +1,34 @@
+import DirectDeposit from '../DirectDeposit';
```

Comment:
The filename has a typo. Should be `direct.deposit.connection.error.cypress.spec.js`